packages/panna-sdk/src/core/onramp/types.ts

```diff
@@ -83,8 +83,10 @@ export interface OnrampPrepareResult {
 
 export type OnrampProvider = 'coinbase' | 'stripe' | 'transak'; // Add more in future
 
+export type ProviderId = OnrampProvider | 'onramp-money';
+
```

Comment:
`OnrampProvider` can be updated to include the new provider. The `ProviderId` type could be removed.

```suggestion
export type OnrampProvider = 'onrampmoney' | 'coinbase' | 'stripe' | 'transak'; // Add more in future
```